docker/build.sh

```diff
@@ -98,10 +98,10 @@ function make_jar()
 
     # make jar.  cache sbt dependencies.
     set +e # Turn off error detection so that opendj has a chance to get stopped before exiting
-    docker run --rm --link postgres:postgres --link $OPENDJ:opendj -e DIRECTORY_URL=$DIRECTORY_URL -e GIT_MODEL_HASH=$GIT_MODEL_HASH -e DIRECTORY_PASSWORD=$DIRECTORY_PASSWORD -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage /working/docker/init_schema.sh /working
+    docker run --rm --link postgres:postgres --link $OPENDJ:opendj -e DIRECTORY_URL=$DIRECTORY_URL -e GIT_MODEL_HASH=$GIT_MODEL_HASH -e DIRECTORY_PASSWORD=$DIRECTORY_PASSWORD -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage:jdk11-2.13.5-1.4.7 /working/docker/init_schema.sh /working
     docker restart $OPENDJ
     sleep 40
-    docker run --rm --link postgres:postgres --link $OPENDJ:opendj -e DIRECTORY_URL=$DIRECTORY_URL -e GIT_MODEL_HASH=$GIT_MODEL_HASH -e DIRECTORY_PASSWORD=$DIRECTORY_PASSWORD -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage /working/docker/install.sh /working
+    docker run --rm --link postgres:postgres --link $OPENDJ:opendj -e DIRECTORY_URL=$DIRECTORY_URL -e GIT_MODEL_HASH=$GIT_MODEL_HASH -e DIRECTORY_PASSWORD=$DIRECTORY_PASSWORD -v $PWD:/working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage:jdk11-2.13.5-1.4.7 /working/docker/install.sh /working
```

Comment:
part of the ticket says:
>We need to make sure that the client libraries build Java 8 and 11 version. 

do we still create the java8 client library?

Reply:
I don't think so, but I am also questioning whether we need to? I think users of the client library should just pin their version? (I think I remember chatting about that?) But maybe we can discuss that.

Reply:
Great question.  I do not recall which service(s) rely on the java8 library nor do I remember if that dependency was just on Rawls or also Sam.  Perhaps we can ask Doug if he knows the answer or maybe Isaac?  We should document whatever we learn and write a ticket that enumerates those services so that when they are upgraded, we can assert that the java8 artifact is no longer needed and we can remove any code for building java8 artifacts.

Reply:
I will ask Doug!

Reply:
WSM, Data Repo, and Agora all use it. WSM has the version pinned, and I am checking Data Repo and Agora, but I _think_ we should be okay.

Reply:
Data Repo and Agora both have the version pinned. Workbench CLI also uses it, and has the version pinned. I think we can safely upgrade, and let all the services know that if they are upgrading, we don't promise Java8 backwards compatibility.

Most of these services are in Java 11 anyway, so that seems like the right approach.

Reply:
this is all set then?  will this theory be tested in dev?

also how are you notifying other services?

Reply:
This is all set, and I am planning on emailing those teams after this goes live to let them know.